ResearchUXFactory/SBASurveyItem+Dictionary.swift

```diff
@@ -33,6 +33,49 @@
 
 import ResearchKit
 
+func key(_ dictionaryKey: DictionaryKey) -> String {
```

Comment:
Does Swift 3 not have a way to call bare C functions like NSStringFromSelector? If not, could you wrap that function in an obj-c helper class to make it available? Or would that not work here for some reason? Or am I missing the point of this enum?